src/fsharp/InfoReader.fs

```diff
@@ -677,6 +687,7 @@ type InfoReader(g: TcGlobals, amap: Import.ImportMap) as this =
     let eventInfoCache = MakeInfoCache GetIntrinsicEventInfosUncached hashFlags1
     let namedItemsCache = MakeInfoCache GetIntrinsicNamedItemsUncached hashFlags2
     let mostSpecificOverrideMethodInfoCache = MakeInfoCache GetIntrinsicMostSpecificOverrideMethodSetsUncached hashFlags0
+    let unionCaseInfoCache = MakeInfoCache GetIntrinsicUnionCaseInfosUncached hashFlags3
```

Comment:
This cache seems to be invalidated every time I make a change in the document (the DU stays the same though), so perhaps it's completely unnecessary?